site/pages/api/delete-consequence.api.ts

```diff
@@ -0,0 +1,42 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { REVIEW_DISRUPTION_PAGE_PATH } from "../../constants";
+import { removeConsequenceFromDisruption } from "../../data/dynamo";
+import { redirectTo, redirectToError } from "../../utils/apiUtils";
+
+const deleteDisruption = async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
+    try {
+        const body = req.body as { id: string | undefined; disruptionId: string | undefined };
```

Comment:
Zod schema can be used to avoid using Type assertion

Reply:
this is not the case for delete disruption as well since it is just an id/2 ids

Reply:
there are other examples within the api that do not use zod where the attributes are minimal